cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -366,4 +366,31 @@ public static String encodeStorageToJson(Storage storage) {
     private static byte[] decodeFromSecret(Secret secret, String key) {
         return Base64.getDecoder().decode(secret.getData().get(key));
     }
+
+    /**
+     * Compares two Secrets with certificates and checks whether any value for a key which exists in both Secrets
+     * changed. This method is used to evaluate whether rolling update of existing brokers is needed when secrets with
+     * certificates change. It separates changes for existing certificates with other changes to the secret such as
+     * added or removed certificates (scale-up or scale-down).
+     *
+     * @param current   Existing secret
+     * @param desired   Desired secret
+     *
+     * @return  True if there is a key which exists in the data sections of both secrets and which changed.
+     */
+    public static boolean didAnyCertificateChangedInSecret(Secret current, Secret desired) {
```

Comment:
I wonder if the name `(do)commonCertificatesDiffer` is better?

Reply:
I think `common` is a bit confusing. I went with `doExistingCertificatesDiffer` - hope that works for you.